addons/jest/src/index.js

```diff
@@ -2,7 +2,7 @@ import addons from '@storybook/addons';
 import deprecate from 'util-deprecate';
 
 const findTestResults = (testFiles, jestTestResults, jestTestFilesExt) =>
-  Array.from(testFiles).map(name => {
+  Array.slice({...testFiles, length: Object.keys(testFiles).length}).map(name => {
```

Comment:
Are you sure it works? `Array.slice` is `undefined` for me

Reply:
I prefered the Array.slice shortcut rather than Array.prototype.slice.call
MDN says there is a full support for Array.slice.
But I think I can replace that with Array.prototype.slice.call

Reply:
Wow, never heard of it. Do you have a link to MDN?

Reply:
here it is! https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice